cluster-controller/src/test/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperationsTest.java

```diff
@@ -200,9 +222,17 @@ private void createCluster(TestContext context, ConfigMap clusterCm) {
                     capturedSs.stream().map(ss -> ss.getMetadata().getName()).collect(Collectors.toSet()));
 
             // Verify that we wait for readiness
-            verify(mockEndpointOps, times(2)).waitUntilReady(any(), any(), anyLong(), anyLong());
-            verify(mockSsOps).waitUntilReady(any(), any(), anyLong(), anyLong());
-            verify(mockPodOps, times(zookeeperCluster.getReplicas())).waitUntilReady(any(), any(), anyLong(), anyLong());
+            verify(mockEndpointOps, times(4)).waitUntilReady(any(), any(), anyLong(), anyLong());
```

Comment:
Why did this change?

Reply:
Because now we are waiting for the endpoints related to the Kafka cluster as well (before we just waited on Zookeeper related endpoints)

Reply:
OK, makes sense.